apps/docs-snippets/src/guide/types/vector.test.ts

```diff
@@ -36,13 +36,12 @@ describe(__filename, () => {
         age: 31,
         salary: 9000,
         idHash: getRandomB256(),
-        ratings: [1, 2, 3],
-        isActive: false,
+        ratings: [5, 6, 7],
+        isActive: true,
```

Comment:
@nedsalk Considering the previous values should work just fine, why was this change necessary?

Reply:
It was for debugging purposes, because the array was the place where the underlying issue happened. I had some console.logs which logged the encoded `ratings` and `isActive` properties, but they were the same in both structs of the test so I changed them for easier discernment of which struct I'm looking at. I think it's better to leave it like this.